### PR TITLE
Added and implemented waitForPageOpen methods

### PR DESCRIPTION
**Pull request by vaspashkevich**
Added and implemented waitForPageOpen methods for Catalog, ItemDetails and Cart pages

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ exports.config = {
   framework: "cucumber",
   cucumberOpts: {
     require: ['./step-definitions/*.js'],
-    timeout: 10000
+    timeout: 60000
```
Comment (panvutka) [inline config.js]: Why do you need so long implicit timeout is you start to use explicit waits in your code?

File: pageObject/CartPage.js [hunk 2]
```diff
@@ -39,6 +39,18 @@ class CartPage extends BasePage {
...
     return await ui5.element.getAllDisplayed(cartSelectors.ItemInCart);
   }
+
+  async waitForPageOpened() {
+    await browser.waitUntil(
+      async function () {
+        return (await ui5.element.isVisible(cartSelectors.proceedBtnSelector));
+      }, {
+        timeout: 5000,
+        timeoutMsg: `Page is not open because element "proceedBtnSelector" is not visible`,
```
Comment (panvutka) [inline pageObject/CartPage.js]: Let's make it more informative. Something like Cart Page has not been loaded. Proceed Btn is not visible`. Please change other messages similar to this one

File: pageObject/CatalogPage.js [hunk 3]
```diff
@@ -1,6 +1,10 @@
 const BasePage = require('../pageObject/BasePage')
 const catalogSelectors = require('../selectors/catalogSelectors.js');
 
+let timeout = 6000;
```
Comment (panvutka) [inline pageObject/CatalogPage.js]: why do you need it here?

**Review by panvutka — CHANGES_REQUESTED** [review]

File: pageObject/CatalogPage.js [hunk 4]
```diff
@@ -17,6 +21,22 @@ class CatalogPage extends BasePage {
...
+        return (await ui5.element.isVisible(catalogSelectors.accesoriesCategory));
+      }, {
+        timeout: 5000,
+        timeoutMsg: `Page is not open because element "accesoriesCategory" is not visible`,
+        interval: 500
+      }
+    );
+ };
+
+  async waitForPageOpenedAssert() {
```
Comment (panvutka) [inline pageObject/CatalogPage.js]: Why do you need 2 different methods for the same action?

**Review by panvutka — CHANGES_REQUESTED** [review]

File: config.js (hunk 1 above)
Comment (vaspashkevich) [reply config.js]: As it is a timeout for the whole test to run, was decided to set it 5 minutes

File: pageObject/CartPage.js (hunk 2 above)
Comment (vaspashkevich) [reply pageObject/CartPage.js]: Done

File: pageObject/CatalogPage.js (hunk 3 above)
Comment (vaspashkevich) [reply pageObject/CatalogPage.js]: Removed

File: pageObject/CatalogPage.js (hunk 4 above)
Comment (vaspashkevich) [reply pageObject/CatalogPage.js]: Only one of theese two versions was used. The other one is now removed

**Pull request merged**